src/Deprecated/Engine/Solution/SolutionWrapperProject.cs

```diff
@@ -755,7 +756,7 @@ static private void AddResolveProjectReferenceTasks
                         catch (Exception e)
                         {
                             if (ExceptionHandling.NotExpectedException(e))
-                               throw;
+                                throw;
```

Comment:
One option that's even nicer:
```suggestion
                         catch (Exception e) when (!ExceptionHandling.NotExpectedException(e))
```

Reply:
I was going to put in a separate PR for exception filters.